public/urls.json

```diff
@@ -1,8 +1,18 @@
 {
-  "edd-ui-claim-status": "https://www.edd.ca.gov/unemployment/claim-status.htm",
-  "edd-ui-eligibility": "https://edd.ca.gov/Unemployment/Eligibility.htm",
+  "ca-gov": "https://www.ca.gov/",
+  "edd-about-accessibility": "https://www.edd.ca.gov/About_EDD/Accessibility.htm",
+  "edd-about-conditions-of-use": "https://www.edd.ca.gov/About_EDD/Conditions_Of_Use.htm",
+  "edd-about-contact": "https://www.edd.ca.gov/About_EDD/Contact_EDD.htm",
+  "edd-about-privacy-policy": "https://www.edd.ca.gov/About_EDD/Privacy_Policy.htm",
+  "edd-ca-gov": "https://edd.ca.gov",
   "edd-covid": "https://www.edd.ca.gov/about_edd/coronavirus-2019/benefit-year-end.htm",
-  "edd-ui-certify": "https://edd.ca.gov/Unemployment/certify.htm",
   "edd-faq-bpo": "https://edd.ca.gov/FAQ_-_Benefit_Programs_Online.htm",
-  "edd-ui-reopen": "https://edd.ca.gov/Unemployment/Reopen-A-Claim.htm"
+  "edd-help-new-claim": "https://uio.edd.ca.gov/UIO/Pages/Public/help/index.htm#t=en-US/Public/NewClaim/UIOnlineNewClaimLandingPage.htm",
```

Comment:
@lomky Just thought of this: Do you think this URL might need to be different depending on mobile vs desktop?

Reply:
I have no way to check - you can't access UIMobile unless you have a real active claim, so I'm not sure if they even have this header